workflow/metrics/scripts/metrics/morans_i.py

```diff
@@ -32,51 +30,16 @@ def get_morans_i_for_category_mean(_ad, _cov, num_mappings=10) -> float:
 
 # Moran's I for a random normal distribution (used as a negative baseline)
 def morans_i_random(adata, output_type, batch_key, label_key, adata_raw, var_key='metrics_features', n_threads=1, **kwargs):
```

Comment:
The only positional parameters are `adata` and `output_type`. Everything else can be removed from the function, if not used, since they will be dealt with by `**kwargs`.

```suggestion
def morans_i_random(adata, output_type, **kwargs):
```